manage.py

```diff
@@ -1,24 +1,23 @@
 #!/usr/bin/env python
 import os
 
-# Edit this if necessary or override the variable in your environment.
-os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'fjord.settings')
-
 from fjord import manage_utils
 
-manage_utils.setup_environ(__file__)
-
-from fjord.base import monkeypatches
-monkeypatches.patch()
 
+if __name__ == "__main__":
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'fjord.settings')
 
-SKIP_CHECK = os.environ.get('SKIP_CHECK', '0')
-USING_VENDOR = os.environ.get('USING_VENDOR', '0')
+    print 'Using %s' % os.environ['DJANGO_SETTINGS_MODULE']
 
+    # This has to get called after DJANGO_SETTINGS_MODULE stuff has
+    # been sorted out.
+    manage_utils.setup_environ()
+    manage_utils.monkeypatch()
 
-if USING_VENDOR == '0' and SKIP_CHECK != '1':
-    manage_utils.check_dependencies()
+    SKIP_CHECK = os.environ.get('SKIP_CHECK', '0')
+    USING_VENDOR = os.environ.get('USING_VENDOR', '0')
```

Comment:
It seems like the default for this used to be `'1'`. Now it is `'0'`. Are you making changes elsewhere in prod to make this ok?